docs/README.md

```diff
@@ -5,46 +5,57 @@ HyP3 documentation is build using [MkDocs](https://www.mkdocs.org/) and the
 
 ## How to
 
-### setting up a development environment
+### Setting up a development environment
 
-From the repository root, you can setup a conda environment with all the 
-necessary dependencies
+In order to automatically document some of our APIs, we use a `conda` environment
+with our APIs installed. You can get Miniconda (recommended) here:
+
+https://docs.conda.io/en/latest/miniconda.html
+
+Once conda is installed, from the repository root, you can create and activate a 
+conda environment with all the necessary dependencies
 
 ```
 conda env create -f conda-env.yml
+conda activate ASFHyP3
 ```
 
-### build and view the documentation site
+Later, you can update the environments dependencies with
```

Comment:
Should this be
the environment's dependencies
?

Reply:
Oops, yes!